catalog/installGit.sh

```diff
@@ -19,8 +19,8 @@ waitForAll
 install "$CATALOG_HOME/github/webhook.js" \
     github/webhook \
     -a feed true \
-    -a description 'Creates a webhook on github to be notified on selected changes' \
-    -a parameters '[ {"name":"username", "required":true, "bindTime":true}, {"name":"repository", "required":true, "bindTime":true}, {"name":"accessToken", "required":true, "bindTime":true},{"name":"events", "required":true} ]' \
+    -a description 'Creates a webhook on GitHub to be notified on selected changes' \
+    -a parameters '[ {"name":"username", "required":true, "bindTime":true, "description": "Your GitHub username"}, {"name":"repository", "required":true, "bindTime":true, "description": "The name of your GitHub repository"}, {"name":"accessToken", "required":true, "bindTime":true, "description": "A webhook or personal token", "doclink": "https://github.com/settings/tokens/new"},{"name":"events", "required":true, "doclink": "https://developer.github.com/webhooks/#events"} ]' \
```

Comment:
As currently implemented, the `username` is the default "organization" when constructing the URL to the repo. However, if the `repository` value contains a `/` then it is interpreted as `<org>/<repo>`. 

This was done to allow the user to create a webhook on a repo that isn't a personal fork, without introducing an explicit new parameter called `organization` (which is unnecessary and confusing for personal forks).

This behavior is probably worth calling out in the `repository` parameter description.